### PR TITLE
Fix #1485: Rich text image is cut-off 

### LINKED ISSUE #1485: Rich text image is cut-off
**Describe the bug**

**To Reproduce**
1. Create Nexus-S 25API emulator
2. Run the app
3. Open Ratios Topic
4. Start `Equivalent Ratios` exploration
5. You will see that the content image is cut-off

**Expected behavior**
The rich text image should never get cut-off.

**Screenshots**
Screenshot_1595010150

**Device**
- Nexus S Emulator
- API 25

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix #1485 
Added `maximum_image_size = 200dp`

Example: Height is 420, width is 440 and `maximumImageSize` is 200
Then `multipleFactor` will be (200/440) = 0.4.
The new height will be 191 and the new width will be 200.

### How to Test
1. Create Nexus-S 25API emulator (any available API level)
2. Run the app
3. Open Ratios Topic
4. Start Equivalent Ratios exploration
5. Try other Explorations also to check if all images are coming correctly on screen

### Screenshot
Nexus S - 28 API(portrait) and 7 WSVGA Tablet(Landscape)
<p>

</p>

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt [hunk 1]
```diff
@@ -88,6 +88,22 @@ class UrlImageParser private constructor(
             drawableHeight = (drawableHeight.toDouble() * multipleFactor).toInt()
             drawableWidth = (drawableWidth.toDouble() * multipleFactor).toInt()
           }
+          val maximumImageSize = context.resources.getDimensionPixelSize(R.dimen.maximum_image_size)
+          if (drawableHeight >= maximumImageSize || drawableWidth >= maximumImageSize) {
+            // The multipleFactor value is used to make sure that the aspect ratio of the image remains the same.
+            // Example: Height is 420, width is 440 and maximumImageSize is 200.
+            // Then multipleFactor will be (200/420).
+            // The new height will be 168 and new width will be 176.
```
Comment (rt4914) [inline utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt]: Actually this logic is incorrect, instead it should be like this:

The multipleFactor value is used to make sure that the aspect ratio of the image remains the same.
// Example: Height is 420, width is 440 and maximumImageSize is 200.
Then multipleFactor will be (200/440).
The new height will be 191 and new width will be 200.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested logic change.

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt (hunk 1 above)
Comment (anandwana001) [reply utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt]: Done, corrected the above screenshot

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt [hunk 2]
```diff
@@ -88,6 +88,22 @@ class UrlImageParser private constructor(
...
             drawableWidth = (drawableWidth.toDouble() * multipleFactor).toInt()
           }
+          val maximumImageSize = context.resources.getDimensionPixelSize(R.dimen.maximum_image_size)
+          if (drawableHeight >= maximumImageSize || drawableWidth >= maximumImageSize) {
+            // The multipleFactor value is used to make sure that the aspect ratio of the image remains the same.
+            // Example: Height is 420, width is 440 and maximumImageSize is 200.
+            // Then multipleFactor will be (200/440).
+            // The new height will be 191 and new width will be 200.
+            val multipleFactor = if (drawableHeight >= drawableWidth) {
+              // If height is less then the multipleFactor, value is determined by height.
```
Comment (rt4914) [inline utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt]: Update this to: 
`// If height is greater then the width, multipleFactor value is determined by height.`

Similarly in line 82 also this was incorrect,
`// If height is less then the width, multipleFactor value is determined by height.`

Ditto for else statement in both cases.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Everything is good except few comments.

**Comment by BenHenning** [comment]
I think we should instead set the maximum based on what we can show in a given orientation & device rather than always using something like 200dp (also, the minimum might need to be larger for tablets since 48dp can be pretty small on tablets).

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt (hunk 2 above)
Comment (anandwana001) [reply utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt]: Done

**Review by rt4914 — CHANGES_REQUESTED** [review]
@anandwana001 Can you just make final change as per Ben's comment for max/min image size?

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**